.travis.yml

```diff
@@ -48,6 +48,7 @@ env:
 
 addons:
   apt_packages:
+    - autoconf-archive
```

Comment:
please add the m4 scripts by copying them to the m4 directory instead of creating a new install dependency (we already have some of the ax scripts there).

Reply:
I pushed the new version with the coverage module copied in the tree. Lets see how it will be handled now in the CI.